src/main/java/com/automation/school/ad_main/Collection/Item.java

```diff
@@ -0,0 +1,47 @@
+package com.automation.school.ad_main.Collection;
+
+public class Item {
+
+    public long id;
+    public String title;
+    public Type type;
+    public static String newTitle;
+    public static Type newType;
+    private static int nextItemId = 0;
+    private double price;
+
+
+    // Create a class constructor for the Item class
+    public Item(long id, String title, Type type) {
+        this.id = id;
+        this.title =title;
+        this.type = type;
+        this.price = price;
```

Comment:
maybe you should include price as a parameter sent to the constructor. As I see now, it seems that the price will always be 0

Reply:
price is included in class public Item(long id, String title, Type type, double price)